Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/RefreshableNode.java

```diff
@@ -75,15 +77,15 @@ protected void onNodeClick(NodeActionEvent e) {
     // to refresh items asynchronously. The default implementation simply
     // delegates to "refreshItems" *synchronously* and completes the Future
     // with the result of calling getChildNodes.
-    protected synchronized void refreshItems(SettableFuture<List<Node>> future, boolean forceRefresh) {
+    protected synchronized void refreshItems(SettableFuture<List<Node>> future) {
         if (!loading) {
             setLoading(true);
             try {
                 removeAllChildNodes();
-                if (forceRefresh) {
-                    refreshFromAzure();
+                if (AuthMethodManager.getInstance().isSignedIn() || this instanceof AzureModule) {
+                    refreshItems();
                 }
-                refreshItems();
```

Comment:
these 2 methods is meaningful in old implementation:
`refreshFromAzure` requires reload the data from Azure(when user click the `fresh` button/menu item).
`refreshItems` indicates we can load from cache(when user click to expand a parent node.)

Reply:
refreshFromAzure  are not used in current impl, remove it and will add it if we need it.